client/www/.env

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_FEEDBACK_APP_ID=5d9c6277-e6ac-42d6-8e51-2354b4870c05
 NEXT_PUBLIC_LLM_EXAMPLE_APP_ID=f22e6525-c977-4879-8d7d-9b3cbeaa7344
+UPTIMEROBOT_API_KEY=REPLACE_ME
```

Comment:
cc @dwwoelfel for thoughts. 

Shall we start hiding out .env? 

This would mean:
* Adding .env to gitignore
* Manually adding these to nextjs' config